spring-cloud-deployer-local/src/main/java/org/springframework/cloud/deployer/spi/local/DockerCommandBuilder.java

```diff
@@ -45,26 +46,32 @@
 	private final Logger logger = LoggerFactory.getLogger(getClass());
 
 	@Override
-	public String[] buildExecutionCommand(AppDeploymentRequest request, Map<String, String> args, Optional<Integer> appInstanceNumber) {
-		List<String> commands = addDockerOptions(request, args, appInstanceNumber);
+	public String[] buildExecutionCommand(AppDeploymentRequest request, Map<String, String> appInstanceEnv,
+										  Map<String, String> appProperties, Optional<Integer> appInstanceNumber) {
+		List<String> commands = addDockerOptions(request, appInstanceEnv, appProperties, appInstanceNumber);
+		// Add appProperties
+		for (String prop : appProperties.keySet()) {
```

Comment:
btw, on windows we don't execute via cmd, java's `ProcessBuilder` uses its own win api's which hopefully have different limitation. This is a common complain on win as low level limits are rather high but then every higher level win api may have their own limits.